code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1091,8 +1091,7 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 	log_admin("[key_name(usr)] sent \"[input]\" as the Tip of the Round.")
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Show Tip")
 
-#define ON_PURRBATION(H) (\H.getorgan(/obj/item/organ/tail/cat) || H.getorgan(/obj/item/organ/ears/cat) || \
-							H.dna.features["ears"] == "Cat" || H.dna.features["human_tail"] == "Cat")
+#define ON_PURRBATION(H) ( iscatperson(H) )

```

Comment:
Why redefine this when you can just replace the 3 things that use it directly underneath with the proper proc?

Reply:
This changes the functionality of ON_PURRBATION.

Reply:
Because I assumed that was what you meant

Reply:
also @ACCount12  I've been repeatedly assured by other coders that it's impossible for a mob to have cat ears without having the DNA flags that iscatperson() checks, because the ears give you the DNA flags when added